recipes/fmt/all/conanfile.py

```diff
@@ -1,11 +1,32 @@
 import os
+import shutil
+
+from conan import ConanFile
+try:
+    from conan.tools.scm import Version
+except ImportError:
+    from conans.tools import Version
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+try:
+    from conan.tools.files import get, apply_conandata_patches
+except ImportError:
+    from conans.tools import get, apply_conandata_patches
+from conan.tools.microsoft.visual import is_msvc, msvc_runtime_flag
+try:
+    from conan.errors import ConanInvalidConfiguration
+except ImportError:
+    from conans.errors import ConanInvalidConfiguration
```

Comment:
all these tools are already in the conan namespace for 1.x, not necessary to do this try: excep: